### PR TITLE
Enables Python 3.10 on CI pipeline

### PR DESCRIPTION
**Pull request by thepabloaguilar**
# Enables Python 3.10 on CI pipeline

This is the first step to start building new features from Python `3.10`, like, Pattern Matching inside `returns`!

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -37,12 +41,14 @@ jobs:
...
     - name: Install dependencies
       run: |
+        poetry config experimental.new-installer false
         poetry config virtualenvs.in-project true
         poetry install
 
         poetry run pip install -U pip
 
     - name: Run tests
+      if: matrix.python-version != '3.10.0-beta.2'
```
Comment (sobolevn) [inline .github/workflows/test.yml:51]: Maybe you can just create a separate step like 

```yml
name: Run lint 
if: matrix.python-version != '3.10.0-beta.2'
run: poetry run flake8 .
```
?

**Review by sobolevn — COMMENTED** [review]
Thanks! Great work!

I will try to release `wps` soon!

File: .github/workflows/test.yml (hunk 1 above)
Comment (thepabloaguilar) [reply .github/workflows/test.yml:51]: It seems more interesting!

Comment (thepabloaguilar) [reply .github/workflows/test.yml]: Done!

**Review by sobolevn — APPROVED** [review]
Thanks a lot!

**Pull request merged**